source/deployment/enterprise-deployment-guide.rst

```diff
@@ -53,9 +53,9 @@ Before You Begin Your Deployment
 What Makes Mattermost Different?
 ********************************
 
-Mattermost is a high-trust messaging platform for enterprise use. As enterprises come in all shapes and sizes, Mattermost has been built to provide an optimal experience for all customers. 
+Mattermost is a high-trust messaging platform for enterprise use. As enterprises come in all shapes and sizes, Mattermost has been built to provide an optimal experience for all customers.
 
-Taking this a step further, the emergence of `ChatOps <https://mattermost.com/chatops>`_ means Mattermost is no longer an exclusive developer tool. In fact, it has the ability to enhance communication through a multitude of `integrations <https://integrations.mattermost.com>`_, extensions, and customization.
+The emergence of `ChatOps <https://mattermost.com/chatops>`_ means Mattermost is no longer an exclusive developer tool. In fact, it has the ability to enhance communication through a multitude of `integrations <https://integrations.mattermost.com>`_, extensions, and customization.
```

Comment:
Would say, "...Mattermost is no longer **a tool predominantly used by developers**. In fact...enhance communication **for many kinds of teams** through a multitude..."